class.json-api-endpoints.php

```diff
@@ -267,11 +267,28 @@ function input( $return_default_values = true, $cast_and_filter = true ) {
 			break;
 		}
 
-		if ( !$cast_and_filter ) {
-			return $return;
+		if ( isset( $this->api->query['force'] ) && 'secure' === $this->api->query['force'] ) {
+			if ( isset( $return['secure_key'] ) ) {
```

Comment:
Why not add this conditional to the compound conditional that encloses it since nothing else happens within it? e.g. 

if ( isset( $this->api->query['force'] ) && 
'secure' === $this->api->query['force'] &&
isset( $return['secure_key'] ) {